.github/workflows/website-deploy.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     tags:
       - v[0-9]+.*
+  release:
```

Comment:
The website is now pulling information from GH releases **upon being built**. The `tags` event happens when we tag the release, which means that latest documentation will be now published, but there will be no release yet on GitHub (as it is manually written and published after the Release workflow finishes). In order for the website to "refresh" when we publish a release, I have added the `release` event, but because we might also happen to delete, unpublish, edit, whatever releases, I think it would be most robust to simply react to everything.

Mind that website deploys workflow is pretty fast, around 2 minutes; most of that being waiting for CDNs to update.

Reply:
@mkaput thanks!